tmt-app/src/Components/Home.js

```diff
@@ -55,13 +63,14 @@ function Home() {
         </FlexContainer>
       </FlexContainer>
 
-      <FlexContainer sx={{ flexDirection: "row" }}>
+      <FlexContainer sx={{ flexDirection: "row", minWidth: "100%" }}>
```

Comment:
Why minWidth over width?

Reply:
I have background images in a container that is positioned absolutely to the parent. However, the parent container width is only 65% (forgot the exact percentage). Therefore, I used minWidth to make it 100% which would allow the child to take up the entire screen when using "justifyContent:  space-between"